### PR TITLE
fix for issue #65

### PR DESCRIPTION
**Pull request by Kotty666**
This fixes Issue #65

### CODE REVIEW

File: templates/varnish.service.erb [hunk 1]
```diff
@@ -41,10 +41,17 @@ ExecStart=/usr/sbin/varnishd <%= scope['::varnish::config::jail_opt'] %> \
   -p <%= k %>=<%= v %> \
 <% end -%>
   -S <%= scope['::varnish::secret_file'] %> \
-  -s <%= scope['::varnish::storage_type'] %>,<% if scope['::varnish::storage_type'] == 'file' -%><%= scope['::varnish::storage_file'] %>,<% end -%><%= scope['::varnish::storage_size'] %> \
-<% scope['::varnish::storage_additional'].each do |k| -%>
+  -s <%= scope['::varnish::storage_type'] %>,<% if scope['::varnish::storage_type'] == 'file' -%><%= scope['::varnish::storage_file'] %>,<% end -%><%= scope['::varnish::storage_size'] -%>
+<% unless [nil, :undefined, :undef, []].include?(scope['::varnish::storage_additional']) -%>
+ \
+<% end %>
```
Comment (NITEMAN) [inline templates/varnish.service.erb:47]: Should be `<% end -%>`

**Comment by fe80** [comment]
Hello @NITEMAN 

It's possible to add some test for this ? You need help ? I can add a default test on the content value on the master if you want

File: templates/varnish.service.erb (hunk 1 above)
Comment (Kotty666) [reply templates/varnish.service.erb]: @NITEMAN  added the missing - didn't see your suggest 2 Years ago :(

**Comment by fe80** [comment]
Ho yes I don't have see they have a part of the same code (I take back the project since 1-2hour)

**Pull request closed without merging**